dprint.h

```diff
@@ -340,49 +353,27 @@ static inline char *dp_log_level_str(int log_level)
 		#define LM_GEN2( _facility, _lev, fmt, ...) \
 			do { \
 				if (is_printable(_lev)){ \
+					DISABLE_FORMAT_WARNING; \
```

Comment:
I added this change because I’m using `%.0s` format specifier to *not* print an integer, which triggers warnings during compilation. Using `%.0s` is the only way to disregard an integer format parameter while retaining the parameter itself.

The reason why this is needed described in the next comment: https://github.com/purecloudlabs/opensips/pull/63/files#r1507641823.